content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -98,6 +98,12 @@ Used on: Pod
 This annotation is used to set [Pod Deletion Cost](/docs/concepts/workloads/controllers/replicaset/#pod-deletion-cost)
 which allows users to influence ReplicaSet downscaling order. The annotation parses into an `int32` type.
 
+{{< note >}}
+Both ingress and egress traffic shaping annotations are experimental feature.
+If you want to enable traffic shaping support, you must add the `bandwidth` plugin to your CNI configuration file (default `/etc/cni/net.d`) and
+ensure that the binary is included in your CNI bin dir (default `/opt/cni/bin`).
+ {{< /note >}}
```

Comment:
I'm afraid we need to include the note **below** the relevant heading and not before it.

Reply:
this note covers both the annotations , so should I place it just after one heading?

Reply:
We can't assume that someone reads about any of the other annotations in a page, so both annotations need at least a brief note. It's OK to hyperlink to another web page (even a README) that includes more detail.